tasks/lambda/project.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "origami-polyfill",
+  "description": "AWS Lambda functions for Polyfill service tasks",
+  "role": "arn:aws:iam::371548805176:role/origami-polyfill_lambda",
```

Comment:
I think so - it's not a credential.  I don't think Apex provides an option to configure roles using env vars.